admin/admin-base.php

```diff
@@ -302,6 +302,15 @@ public function admin_print_footer_scripts() {
 			$params = array_merge( $params, array( 'pll_term_id' => (int) $tag_ID ) );
 		}
 
+		/**
+		 * Filters the list of parameters to add to ajax request.
+		 *
+		 * @since 3.5
+		 *
+		 * @param array $params List of parameters.
+		 */
+		$params = apply_filters( 'pll_ajax_params', $params );
```

Comment:
```suggestion
		 * @since 3.4.5
		 *
		 * @param array $params List of parameters to add to the admin ajax request.
		 */
		$params = apply_filters( 'pll_admin_ajax_params', $params );
```

Reply:
Done in https://github.com/polylang/polylang/pull/1326/commits/4c07b6498b5d308fef80b92c3c756be183d2ef65